pwndbg/lib/arch.py

```diff
@@ -2,8 +2,49 @@
 
 import struct
 import sys
+import typing
 from typing import Literal
 
+# Names of arches that Pwndbg supports
+# The names that GDB/LLDB recognize are converted to a name in this list
+PWNDBG_SUPPORTED_ARCHITECTURES_TYPE = Literal[
+    "x86-64",
+    "i386",
+    "i8086",
+    "mips",
+    "aarch64",
+    "arm",
+    "armcm",
+    "iwmmxt",
+    "rv32",
```

Comment:
```suggestion
    "armcm",
    "rv32",
```
removed `iwmmxt` (probably it is already handled by `arm` name)